platform/cog-platform-fdo.c

```diff
@@ -380,6 +457,29 @@ registry_global (void               *data,
                                          name,
                                          &wl_seat_interface,
                                          version);
+    } else if (strcmp (interface, wl_output_interface.name) == 0) {
+        struct wl_output* output = wl_registry_bind (registry,
+                                             name,
+                                             &wl_output_interface,
+                                             version);
```

Comment:
The indentation of the function arguments in the three last lines looks off. Could you please make sure only spaces are used for indentation?